packages/manager/apps/dedicated/client/app/billing/autoRenew/agreements/user-agreements.routes.js

```diff
@@ -11,6 +11,23 @@ export default /* @ngInject */ (
       template,
       controller: 'UserAccount.controllers.agreements',
       translations: { value: ['.'], format: 'json' },
+      resolve: {
+        gotoAcceptAllAgreements: /* @ngInject */ ($state, atInternet) => (
+          agreements,
+        ) => {
+          atInternet.trackClick({
+            name: 'autorenew::agreements',
+            type: 'action',
+            chapter1: 'popup-agreement',
+          });
+          return $state.go(
+            'app.account.billing.autorenew.agreements.accept-all',
+            {
+              agreements,
+            },
+          );
+        },
+      },
```

Comment:
Valid suggestion @JeremyDec .  But since the QC is almost complete, I would prefer not to make this change now. I will take you point to be followed in my future code, if you are ok with it :)
Thanks a lot for the suggestion